test/projection_math.jl

```diff
@@ -0,0 +1,2 @@
+@test eltype(AbstractPlotting.rotationmatrix_x(1)) == Float64
+@test eltype(AbstractPlotting.rotationmatrix_x(1f0)) == Float32
```

Comment:
Could you wrap this in a `testset`?  Something like:

```julia
@testset "Projection math" begin
    @test eltype(AbstractPlotting.rotationmatrix_x(1)) == Float64
    @test eltype(AbstractPlotting.rotationmatrix_x(1f0)) == Float32 
end

Reply:
I changed it ;)

Reply:
Thanks! I didn't notice the quaternion tests in a test set (maybe I missed it?), so I wasn't sure what to do.

Reply:
They aren't yet, but I have a PR open for that